recipe/meta.yaml

```
@@ -17,22 +18,24 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python {{ python_min }}.*
     - pip
     - poetry
```

```suggestion
    - poetry-core >=1.0.0
```

https://github.com/Mause/duckdb_engine/blob/c67c2e0e4f0d1b805c0917ef8372697541965eff/pyproject.toml#L64